### ISSUE TITLE
Does not install on python 3.8

### ISSUE THREAD

**Issue by VikingScientist** [TheBB/IFEM-to-VT#13]
```
Defaulting to user installation because normal site-packages is not writeable
Collecting IFEM-to-VT
  Downloading IFEM_to_VT-1.0.3-py3-none-any.whl (27 kB)
Collecting lrsplines>=1.5
  Downloading LRSplines-1.7.0-cp38-cp38-manylinux2010_x86_64.whl (5.0 MB)
     |████████████████████████████████| 5.0 MB 3.6 MB/s
Requirement already satisfied: h5py in /home/kjetijo/.local/lib/python3.8/site-packages (from IFEM-to-VT) (2.10.0)
Collecting click
  Downloading click-7.1.2-py2.py3-none-any.whl (82 kB)
     |████████████████████████████████| 82 kB 663 kB/s
ERROR: Could not find a version that satisfies the requirement vtk (from IFEM-to-VT) (from versions: none)
ERROR: No matching distribution found for vtk (from IFEM-to-VT)
```

I suppose it is something wrong with the [vtk-package on pip ](https://pypi.org/project/vtk/), which means that this bug is beyond the scope this repository. Nevertheless we keep it open here so we remember if anyone comes across this later

**Comment by VikingScientist** [comment]
Downloading from https://vtk.org/download/ and installing manually seem to be a workaround

```
wget https://www.vtk.org/files/release/9.0/vtk-9.0.0-cp38-cp38-linux_x86_64.whl .
pip install vtk-9.0.0-cp38-cp38-linux_x86_64.whl
```

**Comment by TheBB** [comment]
VTK has released 9.0.1 on pypi with support for 3.8, so I expect this to be fixed.
